p7c1 init

Artificial-Friend opened the pull request.

Artificial-Friend committed:
p7c1 init

Ketch92 commented on src/main/java/foxtrot/uniform/charlie/kilo/controller/MovieController.java:
```diff
@@ -0,0 +1,45 @@
+        this.responseMapper = responseMapper;
+    }
+
+    @PostMapping("/")
```
no need to add ("/") it's the same with annotation only

Artificial-Friend committed:
p7c2 bugfix:
MovieSessionDao get fixed
refactored:
MovieSession DTOs and mappers
Controller GET annotation added

Artificial-Friend committed:
p7c3 checkstyle

Artificial-Friend committed:
p7c4 fixed: PUT, DELETE

Artificial-Friend committed:
p7c5 MovieSessionRequestDto id field removed

Artificial-Friend committed:
p7c6 refactored
DELETE in MovieSession controller and dao

Artificial-Friend committed:
p7c7 dao changes reverted

boroda4436 commented on src/main/java/foxtrot/uniform/charlie/kilo/model/dto/CinemaHallRequestDto.java:
```diff
@@ -0,0 +1,38 @@
+package foxtrot.uniform.charlie.kilo.model.dto;
+
+public class CinemaHallRequestDto {
+    private Long id;
```
How we can use this property? Shouldn't be the `id` generated by DB?

boroda4436 commented on src/main/java/foxtrot/uniform/charlie/kilo/model/dto/MovieRequestDto.java:
```diff
@@ -0,0 +1,38 @@
+package foxtrot.uniform.charlie.kilo.model.dto;
+
+public class MovieRequestDto {
+    private Long id;
```
same here

Artificial-Friend committed:
p7c8 removed:
id in requests, get(id) in dao
